app/services/webflow_sync/api.rb

```diff
@@ -32,21 +53,32 @@ def create_item(record, collection_slug) # rubocop:disable Metrics/MethodLength
 
     def update_item(record, collection_slug)
       collection = find_webflow_collection(collection_slug)
-      response = client.update_item(
-        { '_cid' => collection['_id'], '_id' => record.webflow_item_id },
-        record.as_webflow_json.reverse_merge(_archived: false, _draft: false), live: true
-      )
+      response = make_request(:update_item, { '_cid' => collection['_id'], '_id' => record.webflow_item_id },
+                              record.as_webflow_json.reverse_merge(_archived: false, _draft: false), live: true)
+
       puts "Updated #{record.inspect} in #{collection_slug}"
       response
     end
 
     def delete_item(collection_slug, webflow_item_id)
       collection = find_webflow_collection(collection_slug)
-      response = client.delete_item({ '_cid' => collection['_id'], '_id' => webflow_item_id })
+      response = make_request(:delete_item, { '_cid' => collection['_id'], '_id' => webflow_item_id })
+      # When record is deleted from application, item is removed from Webflow collection, but item's page stay on live Webflow site.
```

Comment:
nitpick: Let's use `WebFlow` (with capital F) in the docs everywhere, but use `Webflow` (with lowercase F) in the code so that we can have `webflow` in file names instead of `web_flow` as Rails wouldn't understand that correctly.

Reply:
nitpick: but item's page _stays_ on live WebFlow site.

Or even better:
> When we remove an item from WebFlow, its WebFlow page does not immediately get removed (webflowsite.com/item-slug). If we trigger publish for all domains, the page gets immediately removed.

Reply:
Changed, thanks!